src/frontend/src/App.tsx

```diff
@@ -0,0 +1,66 @@
+import { A } from '@solidjs/router'
+
+// Game Advertisemnet Access settings in dropdown menu, battery
```

Comment:
What's with the mix of case? Also "Advertisement"

Reply:
This was a scribbled note. Will cleanup and mark as TODO